.github/workflows/release.yml

```diff
@@ -1,29 +1,25 @@
-name: Go
-
+name: Release
 on:
   push:
-    branches: [ main ]
```

Comment:
also, I think we should move the contents of this file to another file. This file was meant to run whenever any commit was pushed to master.